sections/collage.liquid

```diff
@@ -213,6 +214,23 @@
       "default": "collage",
       "label": "t:sections.collage.settings.mobile_layout.label"
     },
+    {
+      "type": "select",
+      "id": "card_styles",
+      "options": [
+        {
+          "value": "none",
+          "label": "t:sections.collage.settings.card_styles.options__1.label"
+        },
+        {
+          "value": "product-card-wrapper",
+          "label": "t:sections.collage.settings.card_styles.options__2.label"
+        }
+      ],
+      "default": "none",
```

Comment:
Hey team, the impact on defaulting to `none` here will have visual changes for merchants doing manual updates of their theme when we make the next release, is this what you expected? 

Asking just in case I lost context, I came across this while investigating media ratios. Otherwise we would need to have it set to `product-card-wrapper` for Dawn for a smoother merchants' update from what I understood from our last sync. Let me know if I got this wrong! 🙏 

We still have the time to fix this as it's not in merchants' hand yet!

Something unclear for me is if merchants did change the theme settings values for `Cards` on their Dawn theme, I am not sure if this assumption above is still true. 🤔 But if they made changes to the `Media` theme settings, keeping it to `none` [would introduce this](https://screenshot.click/13-05-14lf3-g10ct.png). 

[This is before on Dawn](https://screenshot.click/13-58-p6nbw-cqrqa.png), where all `Collage` blocks are styled from `Cards`, not `Media`. You can see it as I added styling to it on the section below. 

cc. @martinamarien for following up on more explanation next week when we see fit to continue to spread the theme updates knowledge. 🙏 

cc. @sofiamatulis @duygukalaycioglu